src/components/Quiz/Quiz.js

```diff
@@ -1,28 +1,118 @@
+    }
+
+    const fetchSelection = (selection) => {
+        let path;
+        if(questionNum < 2) {
+            path = 'filter.php?i=' + selection;
+        } else {
+            path = 'filter.php?g=' + selection;
+        }
+        const promise = getCocktails(path);
+        promise.then(data => {
+            if (typeof data === 'string' || data instanceof String) {
+                setQuizError(data);
+            } else {
+                filterCocktails(data.drinks);
+            }
+        });
+    }
+
+    const filterCocktails = (cocktails) => {
+        if(questionNum === 0) {
+            setCocktailResults(cocktails);
+        } else {
+            const filtered = cocktailResults.reduce((array, cV) => {
+                cocktails.forEach(cocktail => {
+                    if(cocktail.idDrink === cV.idDrink) {
+                        array.push(cocktail);
+                    }
+                });
+                return array;
+            }, []);
+            setCocktailResults(filtered);
+        }
+        setQuestionNum(questionNum + 1);
+    }
+
+    const pickRandom = () => {
+        const index = Math.floor(Math.random() * cocktailResults.length);
+        const promise = getCocktails('lookup.php?i=' + cocktailResults[index].idDrink);
+        promise.then(data => {
+            if (typeof data === 'string' || data instanceof String) {
+                setQuizError(data);
+            } else {
+                setCocktail(data.drinks[0]);
+            }
+        });
+        setQuestionNum(questionNum + 1);
     }
 
-  return (
-    <div>
-        <NavLink to="/">
-            <p>LOGO - HOME</p>
-        </NavLink>
-        <h2>Pick A Spirit</h2>
-        {makeButtons(spirits)}
-        <h2>Pick An Ingredient</h2>
-        {makeButtons(ingredients)}
-        <h2>Pick A Glass</h2>
-        {makeButtons(glassware)}
-    </div>
-  );
+    return (
+        <div>
+            <NavLink to="/">
+                <p>LOGO - HOME</p>
+            </NavLink>
+            {questionNum === 0 && 
+            <>
+                <h2>Pick A Spirit</h2>
+                {makeButtons(spirits)}
+            </>
+            }
+            {questionNum === 1 && 
+            <>
+                <h2>Pick An Ingredient</h2>
+                {makeButtons(ingredients)}
+            </>
+            }
+            {questionNum === 2 && 
+            <>
+                <h2>Pick A Glass</h2>
+                {makeButtons(glassware)}
+            </>
+            }
+
+            {questionNum === 3 && 
+            <>
+                <button onClick={() => pickRandom()}>See Results</button>
+            </>
+            }
+
+            {questionNum === 4 && 
+            <div className='cocktail-view'>
+                <h2>{cocktail.strDrink}</h2>
+                <img className="drink-img" src={cocktail.strDrinkThumb} />
```

Comment:
Don't forget to add alt text!

Reply:
Just merged that - thanks for the reminder!